src-api/source/includes/_projects.md

```diff
@@ -50,6 +53,8 @@ curl https://circleci.com/api/v1.1/projects?circle-token=:token
 
 Returns an array of all projects you are currently following on CircleCI, with build information organized by branch.
 
+**Note**: When making an API request for Project information, you may experience performance lag and a decrease in overall performance while the request is being processed by the server. To improve performance, CircleCI recommends you pass the `shallow` parameter in your request.
```

Comment:
Could you put this section next to the "recent builds for all projects" endpoint? That's the one that really affects us.

Reply:
Fixed